.claude/skills/proxmox-infrastructure/tools/check_cluster_health.py

```
@@ -0,0 +1,331 @@
+        ipv6_pattern = r'^([0-9a-fA-F]{0,4}:){2,7}[0-9a-fA-F]{0,4}$'
+        return bool(
+            re.match(hostname_pattern, node) or
+            re.match(ipv4_pattern, node) or
+            re.match(ipv6_pattern, node)
+        )
+
+    def run_command(self, command: str) -> str:
+        """Execute command on remote node via SSH"""
+        try:
+            # Use -- to prevent SSH option injection
+            result = subprocess.run(
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)